packages/mesos/extra/dcos-mesos-slave-public.service

```diff
@@ -14,7 +14,9 @@ EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave-public
 EnvironmentFile=-/var/lib/dcos/mesos-slave-common
 EnvironmentFile=/var/lib/dcos/mesos-resources
+EnvironmentFile=-/run/dcos/etc/mesos-slave-public
 ExecStartPre=/bin/ping -c1 ready.spartan
 ExecStartPre=/bin/ping -c1 leader.mesos
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-mesos-agent-public
```

Comment:
Please only use `mesos-slave` or `mesos-agent` for all the internal stuff. I'm fine with either, but don't want to be half way straddling the border. Need to rename the service and make it consistent top to bottom, or keep using slave for now.